### PR TITLE
add wrap_table option 

### LINKED ISSUE #84: Latex tables wrapped in tabular environment
I noticed the changelog that now latex tables are wrapped in a `table` environment. 

This broke my workflow, since I usually want to define the caption inside the latex document and do 

```
\begin{table}
    \caption{This is my caption}
    \label{lab}
    \input{../out/tables/mytable.tex} # starts with \begin{tabular}
\end{table}
```

It doesn't look like there is an option to revert to the old behavior, right? It would be great if this were a keyword argument, like it is in other table packages in other languages. 

Would you be okay with a PR for this?

### PR DESCRIPTION
**Pull request by pdeffebach**
Fixes #84 

I think I've added all the necessary changes. Please let me know!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ PrettyTables.jl Changelog
 Version 0.10.0
 --------------
 
+- ![Enhancement][badge-enhancement] LaTeX tables can
```
Comment (ronisbr) [inline CHANGELOG.md:7]: Please, move this to Version 0.11.0.

File: src/backends/latex/print.jl [hunk 2]
```diff
@@ -144,7 +145,11 @@ function _pt_latex(io::IO, pinfo::PrintInfo;
     # Print LaTeX header
     # ==========================================================================
 
-    if table_type == :tabular
+    if table_type != :tabular && wrap_table == true
```
Comment (ronisbr) [inline src/backends/latex/print.jl:148]: Wouldn't it be better to just ignore `wrap_table` if we are not using `:tabular`? In this version, the user will be forced to always use `table_type = :longtable, wrap_table = false`.

File: src/backends/latex/types.jl [hunk 3]
```diff
@@ -41,6 +42,7 @@ This structure defines the format of the LaTeX table.
     subheader_envs::Vector{String}       = ["texttt"]
     hlines::Vector{Symbol}               = [:begin,:header,:end]
     vlines::Union{Symbol,Vector{Symbol}} = :none
+    wrap_table::Bool                      = true
```
Comment (ronisbr) [inline src/backends/latex/types.jl:45]: I do not think we need this here. IMHO, this is not related to the format of the table, just the user option to wrap or not the table inside the environment `table`.

**Comment by ronisbr** [comment]
Thanks for the PR @pdeffebach ! I left some comments in the source code.

File: src/backends/latex/types.jl (hunk 3 above)
Comment (pdeffebach) [reply src/backends/latex/types.jl:45]: Sorry! I understand better now.

File: src/backends/latex/print.jl (hunk 2 above)
Comment (pdeffebach) [reply src/backends/latex/print.jl:148]: I think it is good to throw an error, actually. 

I just checked in latex and 

```
\begin{table}
\begin{longtable}
... % loooong table
\end{longtable}
\end{table}
```

Doesn't work, as in the long-table doesn't extend to the next page. So allowing this just means that the user gets an error in latex which is really an error in this package.

Comment (ronisbr) [reply src/backends/latex/print.jl:148]: No, I mean, if the user selects `:longtable`, then never add the `table` environment. Something like this, if `table_type` is `:longtable`, then never wrap the table, even if `wrap_table` is `true`.

Comment (ronisbr) [reply src/backends/latex/print.jl:148]: It seems that you only need to delete this `if` that throws an error.

**Comment by ronisbr** [comment]
Another thing to think about: if `wrap_table` is `false`, then should we print the `caption` if a title is available or not?

File: src/backends/latex/print.jl (hunk 2 above)
Comment (pdeffebach) [reply src/backends/latex/print.jl:148]: oh. yeah definitely. Do you want a warning, even?

**Comment by pdeffebach** [comment]
Good question. I would vote to use the caption if a title is given, but not print any `caption` otherwise. Below is a table with two captions. It's weird behavior, but it won't error and the user will be able to understand what's going on. 

Screenshot from 2020-11-08 18-43-29

File: src/backends/latex/print.jl (hunk 2 above)
Comment (ronisbr) [reply src/backends/latex/print.jl:148]: No, it is not necessary. Just your comment that this option does not work with `tabular` is fine :)

Comment (ronisbr) [reply src/backends/latex/print.jl:148]: I plan in the future to add `tabularx`, `tabulary`, etc. In all cases, `warp_table` will work.  `longtable` seems to be the only exception here.

**Comment by ronisbr** [comment]
> Good question. I would vote to use the caption if a title is given, but not print any `caption` otherwise. Below is a table with two captions. It's weird behavior, but it won't error and the user will be able to understand what's going on.

Good! I agree! So always print the caption even if `wrap_table` is `false`. This is handy if I am using PrettyTables to include a table in something like:

```latex
\begin{table}[htb]
% include here
\end{table}
```

In this case, the `\caption` will work.

**Comment by pdeffebach** [comment]
Great. Thanks!

**Comment by ronisbr** [comment]
Perfect! I will merge as soon as the tests pass.
